packages/terra-spacer/src/terra-dev-site/doc/spacer/Spacer.1.doc.jsx

```diff
@@ -1,37 +0,0 @@
-import React from 'react';
-import DocTemplate from 'terra-doc-template';
-import ReadMe from '../../../../docs/README.md';
-
-import { name, version } from '../../../../package.json';
-
-// Example Files
-import SpacerPropsTable from '!terra-props-table-loader!../../../../src/Spacer';
-import SpacerSrc from '!raw-loader!../../../../src/Spacer';
-import Spacer from '../example/SpacerExample';
-import SpacerExampleSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/SpacerExample';
-
-const DocPage = () => (
-  <DocTemplate
-    packageName={name}
-    packageVersion={version}
-    readme={ReadMe}
-    srcPath={`https://github.com/cerner/terra-core/tree/master/packages/${name}`}
-    examples={[
-      {
-        title: 'Spacer',
-        description: 'Spacing default button with a padding value of large+4 and primary button with padding value of large+2',
```

Comment:
We should pull this description into the new mdx file

Reply:
done here https://github.com/cerner/terra-core/pull/2872/commits/4efa60a5a6bd274e0e0926452d27d02ba78d8ad7